46266ae57286a6cb66c87c8c973cf6521766abde
src/test/java/com/yelp/nrtsearch/server/grpc/LuceneServerTest.java
--- 
+++ 
@@ -224,7 +224,8 @@
         "warmer:",
         "  maxWarmingQueries: 10",
         "  warmOnStartup: true",
-        "  warmingParallelism: 1");
+        "  warmingParallelism: 1",
+        "syncInitialNrtPoint: false");
   }
 
   @Test